app/equipment/armor.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any
+
+from app.equipment.equipment import Equipment
+
+
+class Armor(Equipment):
+    def __init__(self, part: str, power: int) -> None:
+        super().__init__(part)
+        self.power = power

```

Comment:
Armour should give protection, not power
![image](https://github.com/user-attachments/assets/b80c54fe-6080-4cd3-a053-6e375c8cc36b)